.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Should we sync the content of these up with the ones in feedback?
https://github.com/dependabot/feedback/edit/master/.github/ISSUE_TEMPLATE/1-bug-report.md
https://github.com/dependabot/feedback/edit/master/.github/ISSUE_TEMPLATE/2-feature-request.md

Reply:
Ah yes nice one 👍